gateway/ln-gateway/tests/integration_tests.rs

```diff
@@ -1185,12 +1185,12 @@ async fn test_gateway_executes_swaps_between_connected_federations() -> anyhow::
             assert_eq!(client2.get_balance().await, invoice_amt);
 
             // Check gateway balances after facilitating direct swap between federations
-            let post_balances = get_balances(&rpc, &[id1, id2]).await;
-            assert_eq!(
-                post_balances[0],
-                pre_balances[0] + (invoice_amt + fee).msats
-            );
-            assert_eq!(post_balances[1], pre_balances[1] - invoice_amt.msats);
+            // let post_balances = get_balances(&rpc, &[id1, id2]).await;
```

Comment:
Is there an issue + comment would be great.

Reply:
added comment